style/javascript.md

```diff
@@ -27,6 +27,7 @@
   * [Do not use async/await or generators](#do-not-use-asyncawait-or-generators)
   * [Do not use Set or Map ](#do-not-use-set-or-map)
   * [Use let and const for new files; do not use var ](#use-let-and-const-for-new-files-do-not-use-var)
+* [Move off Underscore](#move-off-underscore)
```

Comment:
This should go into the 'library rules' section.

Also, most of this audience is written with the point of view that the audience is about to write new code, so 'move off' might be better phrased as "Avoid underscore" or something like that.